packages/nextjs/app/builders/0xC52E3E454a735d4A74F3dd6Ea281F7AebDFAad83/page.tsx

```diff
@@ -0,0 +1,98 @@
+import Image from "next/image";
+import ProfilePic from "./pfp.png";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div className="min-h-screen bg-[#F9F9F9] text-[#333333] flex flex-col items-center justify-center">
+      {/* Profile Picture */}
+      <div className="mb-6">
+        <Image
+          src={ProfilePic}
+          alt="Profile"
+          width={128}
+          height={128}
+          className="rounded-full border-4 border-[#EAEAEA] shadow-lg"
+        />
+      </div>
+
+      {/* Name */}
+      <h1 className="text-4xl font-bold text-[#1A1A1A] mb-4">Cherry 🍒</h1>
+
+      {/* Description */}
+      <p className="text-lg text-[#4F4F4F] max-w-md text-center mb-8">
+        I am currently pursuing a Bachelor&apos;s degree in Computer Science and I aspire to become a blockchain
+        developer and build cool things. Nice to meet you! 🌟
+      </p>
+
+      {/* Social Media Links */}
+      <div className="flex space-x-6 mb-8">
+        <a
+          href="https://github.com/aspiringLilCoder"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="text-[#333333] hover:text-[#1A1A1A]"
+        >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+            stroke="#1A1A1A"
+            stroke-width="2"
```

Comment:
Please remember to check your browser console.  It's currently showing me the following error:

<img width="513" alt="Screenshot 2024-10-29 at 14 58 06" src="https://github.com/user-attachments/assets/020b5272-6176-4188-8c0b-3b3af8dec351">